hail/python/hailtop/aiotools/fs.py

```diff
@@ -218,6 +218,16 @@ def _copy_part_size():
         the destination filesystem is used.'''
         return 128 * 1024 * 1024
 
+    async def copy(self,
```

Comment:
This, oddly, was only defined on a RouterAsyncFS. It's totally FS agnostic though.